packages/graph-editor/src/components/flow/handles.tsx

```diff
@@ -44,7 +44,7 @@ export const HandleContainer = ({
       <Stack
         direction="column"
         gap={2}
-        css={{ flexBasis: full ? '100%' : '50%', position: 'relative', textAlign: type === 'source' ? 'right' : 'left'}}
+        css={{ flexBasis: full ? '100%' : '50%', position: 'relative', textAlign: type === 'source' ? 'right' : 'left', minWidth: '250px' }}
```

Comment:
I'm assuming this is to account for the inline preview?

Reply:
`textAlign` is for inline preview. I don't know exactly what `minWidth` is for. @mck could you help us with the reason for this?